sdk/rust/oak/Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "Apache-2.0"
 oak_abi = "=0.1.0"
 protobuf = "*"
 fmt = "*"
-log = "*"
+log = { version = "^0.4.0", features = ["std"] }
```

Comment:
Why was this needed? We usually don't pin our deps here, we just rely on Cargo.lock for that.

Reply:
This is the dependency that was specified in oak_log. I copied it here as I thought it is needed to avoid incompatibilities. I can change it to "*" if you think it is safe. But I agree, since we track Cargo.lock on git, specifying the version here does not matter.

Reply:
Yes I think we should just use "*" -- as long as it compiles it should be safe (famous last words...). If it does not compile, please feel free to fix whatever is missing inconsistent, or let me know if you think that's not possible.